src/main/java/digital/slovensko/autogram/util/DSSUtils.java

```diff
@@ -29,4 +35,20 @@ public static String buildTooltipLabel(DSSPrivateKeyEntry key) {
         }
         return out;
     }
+
+    public static SignedDocumentValidator getDocumentValidator(CommonDocument document) {
```

Comment:
`get` zvyknem pouzivat len ak to je naozaj getter. toto je skor `build` alebo `create`